gamemode/init.lua

```diff
@@ -88,6 +112,15 @@ function GM:PlayerSilentDeath(ply)
 	end
 end
 
+--- Do not allow respawning by pressing space or clicking unless we only played last round
+function GM:PlayerDeathThink(ply)
+	if(ply.RoundNumber and ply.RoundNumber == self.Round)then
```

Comment:
Also, I used [LuaFormatter](https://github.com/Koihik/LuaFormatter#editor-support) for consistency, please can you format everything with it